gameBase/src/css/styles.css

```diff
@@ -11,8 +11,8 @@ body {
 }
 
 .memory-game {
```

Comment:
try to add 
width: 100%;
  height: calc(100% / 4 * 3); // control aspect ratio 
  padding-bottom: 10px;
  padding-top: 10px;
  padding: 10px;
  max-height: 640px;
  max-width: 640px;

Reply:
in the media query?

Reply:
nope directly in the class .memory-game
the max-width and max-height will prevent that bigger screens expand 100%

Reply:
oh i see!!! added, can you close this pull request so i can make another one